notebooks/Fine_tuning_Code_LLM_on_single_GPU.ipynb

```diff
@@ -0,0 +1,5363 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    tokenizer = AutoTokenizer.from_pretrained(MODEL, use_auth_token=True, trust_remote_code=True)
</pre><p>Let's maybe not use <code>use_auth_token</code> since it's deprecated?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>